anvil/src/main/kotlin/dev/inkremental/Inkremental.kt

```diff
@@ -196,6 +182,27 @@ object Inkremental {
         private val rootView: WeakReference<View> = WeakReference(v)
         internal val iterator = Iterator()
 
+        /** Tags: arbitrary data bound to specific views, such as last cached attribute values  */
+        private val tags: MutableMap<View, MutableMap<String, Any?>> = WeakHashMap()
+
+        operator fun set(v: View, key: String, value: Any?) {
+            var attrs = tags[v]
+            if (attrs == null) {
+                attrs = HashMap()
+                tags[v] = attrs
+            }
+            attrs[key] = value
```

Comment:
Let's kotlinize this while we are here.
`val attrs = tags.getOrPut(v) { HashMap() }`

Reply:
Really nice, totally forgot about it, thanks!  I also shortened get method a bit.